Source/ACE/Entity/GenericObject.cs

```diff
@@ -15,46 +15,24 @@ public GenericObject(AceObject aceObject)
         {
         }
 
-        public GenericObject(ObjectGuid guid, AceObject aceObject)
-            : base(guid, aceObject)
-        {
-        }
+        ////public GenericObject(ObjectGuid guid, AceObject aceObject)
```

Comment:
Also, removing it kills every spawn-able object not explicitly handled by CreateWorldObject which does us no good in the now.